ftw/theming/resources/scss/globals/variables.scss

```diff
@@ -79,6 +79,13 @@ $font-weight-heading: bold !default;
 $max-width-page: 1200px !default;
 $line-height-base: 1.5em !default;
 
+/*
+  Spinner
+  Declare this variable in the policy to define a spinner.
+  Use a resource URL which points to a gif spinner.
+ */
+$spinner-url: "" !default;
```

Comment:
A default should be defined in the theme. An empty `spinner-url` will not break the page.